src/ClientHandle.h

```diff
@@ -453,7 +453,12 @@ class cClientHandle  // tolua_export
 		// TODO: Add Kicking here as well
 	} ;
 	
+#ifdef ANDROID
+#pragma message "Android <atomic> support is incomplete or broken - using plain variable (subject to data races!)"
+	eState m_State;
```

Comment:
This is not an acceptable solution. If we can't use std::atomic, then we need to setup typedefs to a custom std::atomic implementation using cCriticalSection.